esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -169,7 +169,8 @@ void setup()
 
 void loop() 
 {
-    int tnow_ms = millis();
+    unsigned int tnow_ms = millis();
+    static unsigned int last_send = 0;
```

Comment:
could you pl make this a global variable, and init it in setup(), and maybe unsigned long
I guess I would like send_tlast_ms better (see is_connected_tlast_ms)

Reply:
> could you pl make this a global variable, and init it in setup(), and maybe unsigned long I guess I would like send_tlast_ms better (see is_connected_tlast_ms)

I suppose, if you insist, but can you elaborate on why do you want to do it this way?  

Function local static variables occupy the same storage segment and have the same lifetime as global variables, but with scope limited to the function they are declared in.  This helps make clear that they are not and can not be used anywhere else.  Unless setup() is called more than once in the lifetime of the program, the initialization here in the static variable declaration should have exactly the same effect as initializing a global variable in setup().  It is initialized once when the program starts.  Initializing it here, closer to where it is used makes it easier to find the initialization when reading the code.  This use of static function local variables is generally considered best practice for these reasons.  

I'm guessing you already know all of this, but this is why I prefer the static variable.

Reply:
there are also a couple of disadvantages with statis which I'm guessing you also know all, and I do not agree with "is generally considered best practice", that's a matter of who you ask :)
you can keep them as you want, but I'm going to change anyway :)

pl check out if one shouldn't use u long, and consider the renaming

Reply:
Yes, I have no objection to your other requests, just the static.  And I explained why I prefer it.

As I said, if you insist. But I asked why;  What specific disadvantage of the static local approach is motivating your decision to avoid it?  Answering my question will help us avoid wasted time in future PRs

Reply:
this is a good code addition, can we just get it done without drama pl. I sure could spend time to put down muh text, but would it change your opinion, I guess no, so what's the point. Do it or don't do it. :)

Reply:
I'm sorry you consider this drama.  Yes, it might not change my mind, or it might.  Either way, it's your project so I'll do it your way.  But I'd appreciate it if you would do me the courtesy of explaining.  Besides, understanding your reasoning might help me write code you like better in the future with minimum drama.

Reply:
the simple answer would be that I myself have spend too much time and have seen others spending way too much time in finding bugs which at the end were just not reset or re-inited static variables. You put out "is generally considered best practice" which is a pretty mighty and absolutistic claim and I have no intention to convince you otherwise. There is no real relevant advantage of statics but real downsides in both that it is error prone and not compiler efficient. Your statement implies you don't see any disadvantage, and haven't hit the downsides, good for you, but I try to avoid them. I have no itention to produce a list of situations were a static is not only not the best practice but in fact just silly except that in mLRS we have such situations.
btw, I'm pretty sure we have a reset/reinitialization issue in mLRS which I couldn't find yet. I'm surprised the phenomenon hasn't been reported yet. So much on "proper initialization of variables".